sklearn/utils/extmath.py

```diff
@@ -9,17 +9,39 @@
 #          Stefan van der Walt
 #          Kyle Kastner
 #          Giorgio Patrini
+#          Andrew Knyazev
 # License: BSD 3 clause
 
 import warnings
 
 import numpy as np
 from scipy import linalg, sparse
+from scipy import __version__ as scipy__version__
 
 from . import check_random_state
 from ._logistic_sigmoid import _log_logistic_sigmoid
 from .sparsefuncs_fast import csr_row_norms
 from .validation import check_array
+from scipy.sparse.linalg import aslinearoperator, LinearOperator
+
+
+def _parse_version(version_string):
+    version = []
+    for x in version_string.split('.'):
+        try:
+            version.append(int(x))
+        except ValueError:
+            # x may be of the form dev-1ea1592
+            version.append(x)
+    return tuple(version)
```

Comment:
this can be removed in favor of using,

```
rom distutils.version import LooseVersion

if LooseVersion(scipy.__version__) >= "1.3":
....
```

Reply:
fixed in https://github.com/scikit-learn/scikit-learn/pull/12319/commits/ed79fb81128d37e3c421ff437016cc82c4122fd8
But I've just copy-pasted from sklearn\utils\fixes.py 
```
def _parse_version(version_string):
    version = []
    for x in version_string.split('.'):
        try:
            version.append(int(x))
        except ValueError:
            # x may be of the form dev-1ea1592
            version.append(x)
    return tuple(version)


np_version = _parse_version(np.__version__)
sp_version = _parse_version(scipy.__version__)
```

Someone should fix it there for consistency, too. Do you want me to do it? This file sklearn\utils\fixes.py is not currently involved in this PR>